metricbeat/module/system/util.go

```diff
@@ -3,16 +3,12 @@
 package system
 
 import (
-	"math"
 	"runtime"
 
+	"github.com/elastic/beats/libbeat/common"
 	sigar "github.com/elastic/gosigar"
 )
 
-// maxDecimalPlaces is the maximum number of decimal places that the Round
-// function return.
-const maxDecimalPlaces = 4
```

Comment:
Why did you remove this constant and hardcoded it instead?

Reply:
It was requested by @ph to make the number of decimal places configurable. So it was not needed anymore. But I can see having a new exported constant named `DefaultDecimalPlaces` or something similar. It would be cleaner.